<launch>
	<arg name="bagname" default="2022-04-23-14-38-09"/>

	<!-- rviz -->
	<node pkg="rviz" type="rviz" name="rviz" args="-d $(find rosbag_to_pcd)/rviz_config/debug.rviz"/>

	<!-- main -->
	<node pkg="rosbag_to_pcd" type="rosbag_to_pcd" name="rosbag_to_pcd" output="screen">
		<param name="rosbag_path" type="string" value="$(env HOME)/rosbag/airsim/car_1cam_lidar32/$(arg bagname).bag"/>
		<param name="save_dir" type="string" value="$(find rosbag_to_pcd)/save/$(arg bagname)"/>
		<param name="remove_ground" type="bool" value="true"/>
		<param name="grid_dim" type="int" value="100"/>
		<param name="ignore_empty_pc" type="bool" value="false"/>
	</node>
</launch>